src/components/RequestGroupComposition/FormPanel.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <b-container class="p-0">
+    <b-form-row :id="id">
+      <b-col>
+        <b-card no-body>
+          <b-card-header class="p-2">
+            <b-container class="p-0">
+              <b-form-row>
+                <b-col class="text-left">
+                  <i class="align-middle fa-lg mx-2" :class="icon" />
+                  <!-- TODO: The warning and success flicker on page load -->
```

Comment:
It looks like it is. When the component loads, before the first request to the validate API endpoint returns, the form thinks there are no errors so you see a green checkmark indicating there are no errors, but as soon as the validate endpoint returns, you see a red X indicating there are errors. I believe I put this TODO in a while back, years ago at this point... and even though it is still a thing, looking at it again this is probably not that big of a deal and might not even warrant a TODO since I didn't even notice this behavior until you drew my attention to it again, and it takes a bit for the entire form to load anyway.